opensearchpy/_async/client/security.py

```diff
@@ -24,21 +24,19 @@ async def get_account_details(self, params=None, headers=None):
         )
 
     @query_params()
-    async def change_password(
-        self, current_password, password, params=None, headers=None
-    ):
+    async def change_password(self, body, params=None, headers=None):
         """
         Changes the password for the current user.
         """
+        if body in SKIP_IN_PATH:
```

Comment:
All files have some changes in logic compared to the existing code. Can we add tests for these changes?